mii/__init__.py

```diff
@@ -7,3 +7,9 @@
 
 from .config import MIIConfig
 from .grpc_related.proto import modelresponse_pb2_grpc
+
+__version__ = "0.0.0"
+try:
+    from .version import __version__
+except ImportError:
+    pass
```

Comment:
What is the purpose of this?

Reply:
elsewhere the author does

`assert version.parse(mii.__version__) >= version.parse("0.0.3"), "mii must be 0.0.3+"

so this provides a fake version number for versions prior to 0.0.3 that don't have one to allow the assert to work